docs/proposals/kubeedgeadm-scope.md

```diff
@@ -54,137 +56,147 @@ For edge, commands shall be:
+      --kubeedge-version string[="0.3.0-beta.0"]   Use this key to download and use the required KubeEdge version (default "0.3.0-beta.0")
+      --kubernetes-version string[="1.14.1"]       Use this key to download and use the required Kubernetes version (default "1.14.1")
 
-Use "kubeedge cloud [command] --help" for more information about a command.
 ```
 
-### kubeedge cloud init --help
+### kubeedge reset --help
 
 ```
-kubeedge cloud init --help
-<Apt description to be added while implementation>
+kubeedge reset command can be executed in both cloud and edge node
+In cloud node it shuts down the cloud processes of KubeEdge
+In edge node it shuts down the edge processes of KubeEdge
 
 Usage:
-  kubeedge cloud init [flags]
+  kubedge reset [flags]
+
+Examples:
+
+For cloud node:
+kubeedge reset
+
+For edge node:
+kubeedge reset --server 10.20.30.40:8080
+    - For this command --server option is a Mandatory option
 
-Example usage:
-<Apt information to be added while implementation>
 
 Flags:
-  -h, --help                        help for init
-      --kubeedge-version   string   use this key to download and use the required KubeEdge version (Optional, default will be Latest)
-      --kubernetes-version string   use this key to download and use the required Kubernetes version (Optional, default will be Latest)
-      --docker-version     string   use this key to download and use the required Docker version (Optional, default will be Latest)
+  -h, --help            help for reset
+  -s, --server string   IP:Port address of cloud components host/VM
+  
 ```
 
-### kubeedge cloud reset --help
+### kubeedge join --help
 
 ```
-kubeedge cloud reset --help
-<Apt description to be added while implementation>
+
+"kubeedge join" command bootstraps KubeEdge's edge component.
+It checks if the pre-requisites are installed already,
+If not installed, this command will help in download,
+install and execute on the host.
+It will also connect with cloud component to receieve 
+further instructions and forward telemetry data from 
+devices to cloud
 
 Usage:
-  kubeedge cloud reset
+  kubedge join [flags]
 
-Example usage:
-<Apt information to be added while implementation>
+Examples:
 
-Flags:
-  -h, --help   help for reset
-  
-```
+kubeedge join --server=<ip:port>
 
-### kubeedge node join --help
+  - For this command --server option is a Mandatory option
+  - This command will download and install the default version of pre-requisites and KubeEdge
 
-```
-kubeedge node join --help
-<Apt description to be added while implementation>
+kubeedge join --server=10.20.30.40:8080 --docker-version= --kubeedge-version=0.2.1 --kubernetes-version=1.14.1
+kubeedge join --server=10.20.30.40:8080 --docker-version --kubeedge-version=0.2.1 --kubernetes-version=1.14.1
 
-Usage:
-  kubeedge node join [flags]
+  - In case, any option is used in a format like as shown for "--docker-version" or "--docker-version=", without a value
```

Comment:
why should a user type a flag and leave the flag empty for default value ?
User should not be mentioning the flag itself to use default

Reply:
yes if you dont mention the flags default version only will be taken..Just i mentioned in Readme so that user could be familiar about availabe flags !!.

Reply:
if you want me to remove or add another command without any tool flags i can!!

Reply:
I think we can remove this line as we have already mentioned supported flags.

Reply:
ok